packages/fxa-auth-server/test/mail_helper.js

```diff
@@ -62,6 +63,8 @@ module.exports = (printLogs) => {
           } else if (uc) {
             console.log('\x1B[36mUnblock code:', uc, '\x1B[39m');
             console.log('\x1B[36mReport link:', rul, '\x1B[39m');
+          } else if (rpc) {
+            console.log('\x1B[36mReset password Otp:', rpc, '\x1B[39m');
```

Comment:
@chenba similar to the mail helper you have in a separate PR. Should I remove from here and r+ yours?